main.nf

```diff
@@ -25,74 +43,82 @@ if (params.platform == 'illumina' && params.protocol == 'amplicon') {
 } else if (params.platform == 'nanopore') {
     primer_set          = params.primer_set
     primer_set_version  = params.primer_set_version
-    params.artic_scheme = WorkflowMain.getGenomeAttribute(params, 'scheme', log, primer_set, primer_set_version)
+    params.artic_scheme = getGenomeAttribute('scheme', primer_set, primer_set_version)
 }
 
-params.fasta         = WorkflowMain.getGenomeAttribute(params, 'fasta'     , log, primer_set, primer_set_version)
-params.gff           = WorkflowMain.getGenomeAttribute(params, 'gff'       , log, primer_set, primer_set_version)
-params.bowtie2_index = WorkflowMain.getGenomeAttribute(params, 'bowtie2'   , log, primer_set, primer_set_version)
-params.primer_bed    = WorkflowMain.getGenomeAttribute(params, 'primer_bed', log, primer_set, primer_set_version)
+params.fasta         = getGenomeAttribute('fasta')
+params.gff           = getGenomeAttribute('gff')
+params.bowtie2_index = getGenomeAttribute('bowtie2')
+params.primer_bed    = getGenomeAttribute('primer_bed', primer_set, primer_set_version)
 
-params.nextclade_dataset           = WorkflowMain.getGenomeAttribute(params, 'nextclade_dataset'          , log, primer_set, primer_set_version)
-params.nextclade_dataset_name      = WorkflowMain.getGenomeAttribute(params, 'nextclade_dataset_name'     , log, primer_set, primer_set_version)
-params.nextclade_dataset_reference = WorkflowMain.getGenomeAttribute(params, 'nextclade_dataset_reference', log, primer_set, primer_set_version)
-params.nextclade_dataset_tag       = WorkflowMain.getGenomeAttribute(params, 'nextclade_dataset_tag'      , log, primer_set, primer_set_version)
+params.nextclade_dataset           = getGenomeAttribute('nextclade_dataset')
+params.nextclade_dataset_name      = getGenomeAttribute('nextclade_dataset_name')
+params.nextclade_dataset_reference = getGenomeAttribute('nextclade_dataset_reference')
+params.nextclade_dataset_tag       = getGenomeAttribute('nextclade_dataset_tag')
```

Comment:
I feel like the usage of this `getGenomeAttribute` function has changed drasticlally

Reply:
Between the one in the TEMPLATE branch and the one that is now maybe yes. I tried to use the same code that was in the .groovy file so it could work the same, else it won't take all the files needed from the genome.config files properly. I don't know how can this affect future conflicts...